src/Test/Diagnostics/RemoteHost/RemoteHostPanel.xaml.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.CodeAnalysis;
+using Microsoft.Win32;
+
+namespace Roslyn.Hosting.Diagnostics.RemoteHost
+{
+    /// <summary>
+    /// Interaction logic for RemoteHostPanel.xaml
+    /// </summary>
+    public partial class RemoteHostPanel : UserControl
+    {
+        private readonly Workspace _workspace;
+
+        public RemoteHostPanel()
+        {
+            InitializeComponent();
+        }
+
+        public RemoteHostPanel(Workspace workspace) : this()
+        {
+            _workspace = workspace;
+        }
+
+        private async void OnSolutionSave(object sender, RoutedEventArgs e)
```

Comment:
:grey_exclamation: `async void` will be flagged as warnings in the build soon. It will save effort to go ahead and remove them now.

Reply:
those xaml page is a tool to test serialization. not part of a product or a test or anything. it is in diagnostic project which is a place where we put various tools to test features while developing.

Reply:
sure.